src/test/java/eu/yals/test/ui/HomePageTest.java

```diff
@@ -6,24 +6,27 @@
 import static org.junit.Assert.fail;
 
 /**
- * Base for all tests testing HomePage aka /
+ * Base for all tests testing HomePage aka /.
  *
  * @since 2.7
  */
 public abstract class HomePageTest extends VaadinTest {
-  protected HomeViewPageObject homeView;
+    protected HomeViewPageObject homeView;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'homeView' must be private and have accessor methods.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)